extensions/Vendor/EXT_texture_transform/README.md

```diff
@@ -0,0 +1,82 @@
+# EXT_texture_transform
+
+## Contributors
+
+* Steven Vergenz, Microsoft ([steven.vergenz@microsoft.com](mailto:steven.vergenz@microsoft.com))
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+Many techniques can be used to optimize resource usage for a 3d scene. Chief among them is the ability to minimize the number of textures the GPU must load. To achieve this, many engines encourage packing many objects' low-resolution textures into a single large texture atlas. The region of the resulting atlas that corresponds with each object is then defined by vertical and horizontal offsets, and the width and height of the region.
+
+To support this use case, this extension adds `offset` and `scale` properties to textureInfo structures. These properties would typically be implemented as transforms on the UV coordinates. In GLSL: `vec2 uvTransformed = Scale * Uv + Offset;`
+
+This is equivalent to Unity's `Material#SetTextureOffset` and `Material#SetTextureScale`, or Three.js's `Texture#offset` and `Texture#repeat`.
+
+## glTF Schema Updates
+
+The `EXT_texture_transform` extension may be defined on `textureInfo` structures. It may contain the following properties:
+
+| Name       | Type       | Default      | Description
+|------------|------------|--------------|---------------------------------
+| `offset`   | `array[2]` | `[0.0, 0.0]` | The offset of the UV coordinate origin as a percentage of the texture dimensions.
```

Comment:
Just a minor note: I believe offset is not a percentage, correct? It is encoded as a factor relative to the texture dimensions.

Reply:
Oh, good catch. I'm used to percentages being [0,1)